spec/actors/hyrax/actors/collections_membership_actor_spec.rb

```diff
@@ -16,12 +16,16 @@
   end
 
   describe 'the next actor' do
+    let(:collection_type) { create(:collection_type) }
+    let(:collection) { create(:collection, collection_type_gid: collection_type.gid) }
```

Comment:
This collection_type isn't used elsewhere in the file. Would it make sense to move this to factory, so that it automatically creates a type if you don't provide one?